.coveragerc

```diff
@@ -179,6 +179,9 @@ omit =
     homeassistant/components/tesla.py
     homeassistant/components/*/tesla.py
 
+    homeassistant/components/thethingsnetwork.py
+    homeassistant/components/*/thethingsnetwork*.py
```

Comment:
Platforms need to have same name as component

Reply:
Technically it seems to work. Haven't check for side effects.

Too bad, that a component can't be re-used for multiple implementations for the same platform. I was thinking about `thethingsnetwork_storage`, `thethingsnetwork_mqtt` or `thethingsnetwork_http`.

Reply:
I think that we should consolidate on one implementation.

Reply:
OK